component/main.jsonnet

```diff
@@ -1,10 +1,101 @@
 // main template for openshift4-olm
+local com = import 'lib/commodore.libjsonnet';
+local espejote = import 'lib/espejote.libsonnet';
 local kap = import 'lib/kapitan.libjsonnet';
 local kube = import 'lib/kube.libjsonnet';
+local olm = import 'lib/openshift4-olm.libsonnet';
 local inv = kap.inventory();
+
 // The hiera parameters for the component
 local params = inv.parameters.openshift4_olm;
 
+local patchDisableDefaultSources = [
+  espejote.managedResource('disable-default-sources', 'openshift-marketplace') {
```

Comment:
A tiny quick description in the form of https://github.com/appuio/component-openshift4-nodes/blob/7622ff25731f67177b0e844c22688270196d5dc5/component/autoscaler.jsonnet#L15-L23 would be nice.